Building the website locally fails with npm

__Feature to document__

<!-- Name of the feature(s) and a link to related pull request of the feature implementation. -->
I was following the instructions for getting the site setup locally in https://github.com/webpack/webpack.js.org/blob/8d4b9a0617a57c2175ffcfbaded99c5f3448521e/.github/CONTRIBUTING.md, but I was hitting issues with npm. `npm install` worked fine, but got the following errors from running `npm start`:

```
$ npm run start

> webpack.js.org@0.1.2 start /Users/richardx/git/webpack.js.org
> npm run clean-dist && cross-env NODE_ENV=development webpack serve --config webpack.dev.js --env dev --progress


> webpack.js.org@0.1.2 clean-dist /Users/richardx/git/webpack.js.org
> rimraf ./dist

<i> [webpack-dev-server] Project is running at http://[::]:3000/
<i> [webpack-dev-server] Content not from webpack is served from /Users/richardx/git/webpack.js.org/dist
<i> [webpack-dev-server] 404s will fallback to /index.html
<e> [webpack-dev-middleware] assets by path *.js 6.1 MiB 129 assets
<e> assets by info 175 KiB [immutable]
<e>   assets by chunk 56.4 KiB (auxiliary name: index)
<e>     assets by path *.svg 9.79 KiB 7 assets
<e>     assets by path font/ 33.2 KiB 2 assets
<e>     assets by path *.png 13.3 KiB
<e>       asset icon-square-small-slack.9281492bb267314634b4.png 8.03 KiB [emitted] [immutable] [from: assets/icon-square-small-slack.png] (auxiliary name: index)
<e>       asset ag-grid-logo.4fe9d34532b2c0821fc9.png 5.3 KiB [emitted] [immutable] [from: assets/ag-grid-logo.png] (auxiliary name: index)
<e>   assets by path *.css 101 KiB
<e>     asset index.d414e50dfac9c36147bf.css 99.5 KiB [emitted] [immutable] (name: index) 1 related asset
<e>     asset components_Support_Support_jsx.71e5f8382e26c173d59d.css 1.52 KiB [emitted] [immutable] 1 related asset
<e>   asset segment.0a9ec28e60cc1d77c8f6.png 17.7 KiB [emitted] [immutable] [from: components/Support/logos/segment.png]
<e> asset favicon.ico 31.3 KiB [emitted]
<e> asset index.html 362 bytes [emitted]
<e> Entrypoint index 2.74 MiB (2.68 MiB) = index.d414e50dfac9c36147bf.css 99.5 KiB index.bundle.js 2.64 MiB 13 auxiliary assets
<e> cached modules 5.54 MiB (javascript) 39.3 KiB (asset) 101 KiB (css/mini-extract) [cached] 577 modules
<e> runtime modules 41.6 KiB 19 modules
<e> javascript modules 5.89 KiB 17 modules
<e>
<e> ERROR in ./components/Support/Support.jsx 51:0-41
<e> Module not found: Error: Can't resolve './_supporters.json' in '/Users/richardx/git/webpack.js.org/src/components/Support'
<e>  @ ./components/Splash/Splash.jsx 36:9-37
<e>  @ ./components/Site/Site.jsx 43:0-38 250:21-27
<e>  @ ./index.jsx 17:0-42 30:33-37
<e>
<e> webpack 5.11.1 compiled with 1 error in 6040 ms
<i> [webpack-dev-middleware] Failed to compile.
<i> [webpack-dev-middleware] Compiling...
<e> [webpack-dev-middleware] assets by status 3.63 MiB [cached] 142 assets
<e> assets by status 2.65 MiB [emitted]
<e>   assets by path *.js 2.65 MiB
<e>     asset index.bundle.js 2.64 MiB [emitted] (name: index) 1 related asset
<e>     asset index.17005ca5d5212a45c55d.hot-update.js 12.6 KiB [emitted] [immutable] [hmr] (name: index) 1 related asset
<e>   asset index.html 362 bytes [emitted]
<e>   asset index.17005ca5d5212a45c55d.hot-update.json 29 bytes [emitted] [immutable] [hmr]
<e> Entrypoint index 2.75 MiB (2.68 MiB) = index.d414e50dfac9c36147bf.css 99.5 KiB index.bundle.js 2.64 MiB index.17005ca5d5212a45c55d.hot-update.js 12.6 KiB 14 auxiliary assets
<e> cached modules 5.54 MiB (javascript) 39.3 KiB (asset) 101 KiB (css/mini-extract) [cached] 577 modules
<e> runtime modules 41.6 KiB 19 modules
<e> javascript modules 5.89 KiB 17 modules
<e>
<e> ERROR in ./components/Support/Support.jsx 51:0-41
<e> Module not found: Error: Can't resolve './_supporters.json' in '/Users/richardx/git/webpack.js.org/src/components/Support'
<e>  @ ./components/Splash/Splash.jsx 36:9-37
<e>  @ ./components/Site/Site.jsx 43:0-38 250:21-27
<e>  @ ./index.jsx 17:0-42 30:33-37
<e>
<e> webpack 5.11.1 compiled with 1 error in 3610 ms
<i> [webpack-dev-middleware] Failed to compile.
```

I even attempted to manually run `npm run fetch`, but still got these errors:

```
$ npm run fetch

> webpack.js.org@0.1.2 fetch /Users/richardx/git/webpack.js.org
> run-p fetch:*


> webpack.js.org@0.1.2 fetch:readmes /Users/richardx/git/webpack.js.org
> node src/utilities/fetch-package-readmes.js


> webpack.js.org@0.1.2 fetch:supporters /Users/richardx/git/webpack.js.org
> node src/utilities/fetch-supporters.js

(node:20417) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/Users/richardx/git/webpack.js.org/repositories/loaders.json'
(node:20417) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:20417) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

At this point, I gave up and switched to using `yarn`, since I saw the note in `CONTRIBUTING.md` that said that the core team primarily uses it. `yarn` worked perfectly for me.

If it's too much trouble to fix the setup for npm, then I'd suggest just updating CONTRIBUTING.md to say that yarn is required for developing locally, since it's better to have instructions that work for yarn, even if yarn may be less familiar to developers than npm.

OS/Tool versions:

- macOS 10.15.7 (Catalina)
- Node.js v10.18.1
- npm 6.13.4

__Author(s)__

<!-- Mention feature author(s) in order to know who is assignable for review outside of the documentation team. -->

__Additional information__

<!-- Please provide any additional information that is required in order to document the feature in the best possible way or say N/A if linked PR has more than enough details. -->

[ ] I am willing to work on this issue and submit a pull request.

I once tried to solve the `Module not found: Error: Can't resolve './_supporters.json'` issue here https://github.com/webpack/webpack.js.org/pull/4155, but it failed.

Actually I'm a little surprised that `npm` would work as we have `resolutions` field https://github.com/webpack/webpack.js.org/blob/master/package.json#L165 which I believe is yarn only.

I agree it's better to update `CONTRIBUTING.md`.

--- a/.github/CONTRIBUTING.md
+++ b/.github/CONTRIBUTING.md
@@ -11,27 +11,26 @@ page please create an issue first.
 Start by ensuring that you have Node.js installed and forking the repository:
 
 - Install [Node.js][1] if you have not already.
+- Install [Yarn][13] if you have not already. Yarn is an alternative to npm, and it is required for building the site.
 - Fork the **webpack.js.org** repo from [the main repository][2].
 - `git clone <your-clone-url> && cd webpack.js.org`
 
 Once you are in the project directory, run the following commands:
 
-- `npm install` to pull all dependencies. Make sure to check notes below if you prefer Yarn.
-- `npm run build` to create a production version of the site.
-- `npm start` to develop on a local webpack-dev-server: [localhost:3000][3].
+- `yarn` to pull all dependencies.
+- `yarn build` to create a production version of the site.
+- `yarn start` to develop on a local webpack-dev-server: [localhost:3000][3].
 
-> NOTE: run `npm fetch` before running `npm start` command for the first time
+> NOTE: run `yarn fetch` before running `yarn start` command for the first time
 
-- `npm run fetch` to retrieve external documentation/data.
+- `yarn fetch` to retrieve external documentation/data.
 
-The final command, `npm run fetch`, is optional as both `npm run build` and `npm start`
+The final command, `yarn fetch`, is optional as both `yarn build` and `yarn start`
 will do this automatically. This step pulls in documentation for loaders/plugins hosted
 in separate repositories such as the ones found in the [webpack-contrib][4] organization.
 See the `package.json` for the full list of `scripts`.
 
 > Note that __Node 6.9.x__ or above is required for the build process to run properly.
-
-> Although __Yarn__ is not required to work with webpack docs, the core team is using it more extensively. If you prefer to use Yarn make sure that you [get it installed][13] and run `yarn` to pull all dependencies, `yarn build` to build the project, or `yarn start` to start the development server.
 
 > On Debian and Ubuntu operating systems you may have to use `node >= 7.0.0` to avoid build errors with `node-sass`. Please note that we don't officially support building on these systems.
 

